web-app/package.json

```
@@ -3,16 +3,21 @@
   "description": "Pub Golf Web App",
   "version": "0.0.1",
   "scripts": {
-    "tailwind": "npx tailwind build static/tailwind.config.css -o static/tailwind.css",
-    "protoc": "protoc -I=../proto pubgolf.proto --js_out=import_style=commonjs:./src/proto --grpc-web_out=import_style=commonjs,mode=grpcwebtext:./src/proto",
-    "dev": "sapper dev",
-    "build": "sapper build",
-    "export": "sapper export",
-    "start": "node __sapper__/build",
+    "build": "run-s protoc sapper:build",
     "cy:run": "cypress run",
     "cy:open": "cypress open",
-    "test": "run-p --race dev cy:run",
-    "postinstall": "run-p tailwind protoc"
+    "dev": "run-s protoc sapper:dev",
+    "export": "sapper export",
+    "postinstall": "run-p tailwind protoc",
+    "prettier": "prettier --write ./src/**/*.{js,svelte}",
+    "protoc": "run-s protoc:clean protoc:build",
+    "protoc:build": "protoc -I=../proto pubgolf.proto --js_out=import_style=commonjs:./src/proto --grpc-web_out=import_style=commonjs,mode=grpcwebtext:./src/proto",
```

I think this needs to be updated to the new command you put in the README
```suggestion
    "protoc:build": "protoc -I=../proto ../proto/*.proto --js_out=import_style=commonjs:./src/proto --grpc-web_out=import_style=commonjs,mode=grpcwebtext:./src/proto",
```

---

Oops